lib/Github/Api/Integrations.php

```diff
@@ -26,4 +26,52 @@ public function createInstallationToken($installationId, $userId = null)
 
         return $this->post('/installations/'.rawurlencode($installationId).'/access_tokens', $parameters);
     }
+
+    /**
+     * Find all installations for the authenticated integration.
+     *
+     * @return array
+     */
+    public function findInstallations()
+    {
+        return $this->get('/integration/installations');
+    }
+
+    /**
+     * List repositories that are accessible to the authenticated installation.
+     *
+     * @param int $userId
+     *
+     * @return array
+     */
+    public function listRepositories($userId)
```

Comment:
No if you call the api with the userId you just get a 403 error irc. The repositories are also always cehcked for the current installation and the repo's the user has access to. I guess github will change this api route in the future so the userId is part of the uri.